frontend/src/scenes/data-warehouse/editor/multitabEditorLogic.tsx

```diff
@@ -742,6 +754,33 @@ export const multitabEditorLogic = kea<multitabEditorLogicType>([
             const allModelQueries = await getStorageItem(editorModelsStateKey(props.key))
             const activeModelUri = await getStorageItem(activeModelStateKey(props.key))
 
+            const storedAllTabs = await getStorageItem(`${props.key}/allTabs`)
+            const storedInProgressViewEdits = await getStorageItem(`${props.key}/inProgressViewEdits`)
+
+            if (storedAllTabs) {
+                try {
+                    const parsedTabs = JSON.parse(storedAllTabs) as QueryTab[]
+                    actions.setTabs(parsedTabs)
+                } catch (error) {
+                    posthog.capture('sql-editor-alltabs-parse-failure', { error })
```

Comment:
We should be using `posthog.captureException(..)` here instead so that these go into error tracking